app/src/main/java/com/futurice/freesound/feature/common/ui/adapter/RecyclerViewAdapter.java

```diff
@@ -121,7 +121,7 @@ private void updateAllItems(@NonNull final List<DisplayableItem> items) {
     }
 
     /**
-     * Do not use for first update of the adapter.
+     * Only used for first update of the adapter.
```

Comment:
This is wrong since the idea is that this method is never used for the first update. Was the previous wording incorrect? 

> Do not use for first update of the adapter